src/collective/volto/formsupport/restapi/services/submit_form/post.py

```diff
@@ -276,8 +282,25 @@ def send_data(self):
             self.send_mail(msg=msg, charset=charset)
 
     def prepare_message(self):
+        template_name = self.email_format_page_template_mapping[
+            self.block.get("email_format", "")
```

Comment:
probably better like this:

```
email_format = self.block.get("email_format", "")
template_name = self.email_format_page_template_mapping.get(email_format, "send_mail_template")
```

So you don't need to set a "" key in mapping dict